### PR TITLE
Update README per cd03959116f7b27ab3ec7ccace98bd10be435397

### PR DESCRIPTION
**Pull request by psoloway**
The `force`, `warn`, and fragment `ensure` parameters have been added back and deprecated. Updating README to reflect this.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -170,6 +174,10 @@ Specifies whether to overwrite the destination file if it already exists. Valid
 
 Specifies a validation command to apply to the destination file. Requires Puppet version 3.5 or newer. Valid options: a string to be passed to a file resource. Default value: undefined.
 
+#####`warn`
+
+Deprecated as of concat v2.0.0. Has no effect. Please use `warn_header` instead.
```
Comment (bmjen) [inline README.md]: We've removed warn_header completely. 'warn' was readded to replace warn_header.

**Comment by electrical** [comment]
Would you mind editing the link to the library in the 'What concat affects' part as well while you are working on it :-)
Should link to https://forge.puppetlabs.com/electrical/file_concat

Thnx

**Comment by bmjen** [comment]
Also, do you know how to squash these commits back down to the original README update commit?

**Comment by psoloway** [comment]
Yep. Shall I do that now?

**Comment by psoloway** [comment]
Confirmed via HipChat, and squashed.

**Pull request merged**